ignite/services/network/networktypes/genesisinformation.go

```diff
@@ -151,6 +169,10 @@ func (gi *GenesisInformation) RemoveGenesisValidator(address string) {
 	}
 }
 
+func (gi *GenesisInformation) AddParamChange(pc ParamChange) {
+	gi.ParamChanges = append(gi.ParamChanges, pc)
```

Comment:
We should remove the previous value from indexation `moduleName-paramName` if already exists

Another solution is to use a `map`, compared to `GenesisAccount` their shouldn't be determinism issues given the end genesis is the same independently of the order of iteration

EDIT: unless it is a precondition that there is no duplicated indexes given that we fetch data from the chain

Reply:
Yes, my understanding is that on chain, they are written to the same index, so for any given invocation of this function, each `moduleName-paramName` index is modified once.